en/faq.md

```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
 
 [[toc]]
 
-## 站点使用相关
+## Station usage related
 
-### 收不到验证邮件？
+### Can't receive verify email？
 
-请先检查你的邮箱的垃圾箱。如果垃圾箱中也没有验证邮件，尝试再发送一次验证邮件。
+Please check your email's trash can at first. If you can't find verify email in trash can, please try send verify email again.
 
-如果仍然收不到验证邮件，请检查你的账号绑定的邮箱是否正确，如果不正确则将其更改为正确的邮箱。
+If you still can't receive verify email, please check your email that bind your account is correct, if incorrect, please change it to your correct email.
```

Comment:
If you still cannot receive verification email, please check your whether email address that bind your account is correct. If not, please change it to the correct one.